lib/cli/src/warn.ts

```diff
@@ -0,0 +1,22 @@
+import globby from 'globby';
+import { logger } from '@storybook/node-logger';
+
+interface Options {
+  hasTSDependency: boolean;
+}
+
+export const warn = ({ hasTSDependency }: Options) => {
+  if (!hasTSDependency) {
+    const hasTSFiles = !!globby.sync(['**/*.(ts|tsx)', '!**/node_modules', '!**/*.d.ts']).length;
```

Comment:
`'**/*.(ts|tsx)'` is not a proper glob 😄   See https://github.com/storybookjs/storybook/pull/10926

Reply:
This is what I used in codemod if it's useful:

https://github.com/storybookjs/storybook/blob/next/lib/codemod/src/index.js#L45

Reply:
According to the docs here:
https://github.com/isaacs/minimatch#usage
https://github.com/sindresorhus/globby#patterns

This is supported, and is called `extglob`. Maybe you can give me some more info around the change? I didn't see any docs linked in the PR referenced.

Reply:
`const hasTSFiles = !!globby.sync(['**/*.(ts|tsx)', '!**/node_modules', '!**/*.d.ts']).length;`

should be

`const hasTSFiles = !!globby.sync(['**/*.@(ts|tsx)', '!**/node_modules', '!**/*.d.ts']).length;`